src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -61,7 +61,7 @@ const PageLayoutWrapper = ({
   let wordCount: wordCountType = 0;
   let propsForOJExperiment = {};
   const experimentVariant = useOptimizelyMvtVariation(
-    OPTIMIZELY_CONFIG.flagKey,
+    OPTIMIZELY_CONFIG.ruleKey,
```

Comment:
Is there a way of doing a look up of the `isClientSide` value https://github.com/bbc/simorgh/blob/82706e5d13bc87f0f386b98a68ee717b3d248439/src/app/hooks/useOptimizelyVariation/index.jsx#L5, and then choosing the correct config key?

Reply:
Its using the `useOptimizelyMvtVariation` rather than the `useOptimizelyVariation` hook. I think that `isClientSide` value is more of a fail safe mechanism so that the `useOptimizelyVariation` hook cannot ever run when its set to `false`.

I dare say we could re-architect some of these hooks/methods to make them more seemless across server/client side mvt.

Reply:
The flag/rule key stuff is confusing. There are instances on the client side where `ruleKey` is used as well, so its not just a server-side only thing.

Reply:
Could maybe just not allow values to be passed into these hooks. The MVT one always uses `ruleKey` and the non-MVT one always uses `flagKey`

Reply:
Uhh these flags and rules make it so confusing, the old way was better

Reply:
There is a ticket to look into re-architecting the hooks https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-16